src/Http/Controllers/AddressController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+namespace Tychovbh\Mvc\Http\Controllers;
+
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+use Tychovbh\Mvc\Http\Controllers\AbstractController;
+
+class AddressController extends AbstractController
+{
+    /**
+     * @param Request $request
+     * @return JsonResource
+     */
+    public function store(Request $request): JsonResource
+    {
+        $zipcode = $request->has('zipcode') ? $request->get('zipcode') : '';
```

Comment:
zipcode and house_number are no required no need to check if the request has them:
$zipcode= $request->get('zipcode');